tests/container/vnc-script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# vnc-script
+#
+# Copyright (C) 2016 Intel Corporation
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+set -e
+set -x
+# this runs inside the container. Returns 0 on success.
+ls -al /workdir
+# make a vncuser from the skeleton
+useradd -m --skel=/etc/vncskel vncUser
+
+# install xdpyinfo in the distros that are missing it
+if [ -x "$(command -v dnf)" ]; then
```

Comment:
I did it this way in another file https://github.com/crops/yocto-dockerfiles/blob/master/build-install-dumb-init.sh#L6. You don't have to do it that way, but elif seems easier to follow. I'm assuming that ubuntu has xdpyinfo installed by default?